lib/utils/screen-manager.js

```diff
@@ -108,10 +97,21 @@ ScreenManager.prototype.done = function () {
   this.rl.output.write('\n');
 };
 
-function breakLines(lines) {
+ScreenManager.prototype.normalizedCliWidth = function () {
+  var width = cliWidth({
+    defaultWidth: 80,
+    output: this.rl.output
+  });
+  if (process.platform === 'win32') {
+    return width - 1;
+  }
+  return width;
+};
+
+ScreenManager.prototype.breakLines = function (lines) {
```

Comment:
do you mean the output or really just the returning value of `cliWidth()`? :confused: anyways `this.rl.output` is the reason I had to turn them into class methods

The first thing I tried was sending the output as a parameter but then it cascades to `breakLines()` and `forceLineReturn()` having to add an extra parameter to them just to send down `this.rl.output` - it was a much uglier solution to be honest

I'm pretty sure it's going to look the same thing if we were going to do the same thing with the `width`